src/librustc_codegen_llvm/back/lto.rs

```diff
@@ -122,7 +120,7 @@ fn prepare_lto(cgcx: &CodegenContext<LlvmCodegenBackend>,
                 let bc = SerializedModule::FromRlib(bc);
                 upstream_modules.push((bc, CString::new(id).unwrap()));
             }
-            timeline.record(&format!("load: {}", path.display()));
+            cgcx.profile(|p| p.generic_event(format!("load: {}", path.display())));
```

Comment:
`start-end` event spanning the loop body of `for &(cnum, ref path) in cgcx.each_linked_rlib_for_lto.iter() { ...`